README.md

```diff
@@ -440,39 +471,23 @@ Use [`create` command](/docs/RectorRecipe.md).
 - [PHP Parser Nodes Overview](/docs/NodesOverview.md)
 - [Generate Rector from Recipe](/docs/RectorRecipe.md)
 
-## How to Contribute
-
-Just follow 3 rules:
-
-- **1 feature per pull-request**
-- **New features need tests**
-- Tests, coding standards and PHPStan **checks must pass**:
-
-    ```bash
-    composer complete-check
-    ```
-
-    Do you need to fix coding standards? Run:
-
-    ```bash
-    composer fix-cs
-    ```
-
-We would be happy to accept PRs that follow these guidelines.
-
 ## Run Rector in Docker
 You can run Rector on your project using Docker:
```

Comment:
There should be at least link to separated file

Reply:
Done.